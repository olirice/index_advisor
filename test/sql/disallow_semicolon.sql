begin;

    create extension index_advisor cascade;

    select index_advisor($$ select 1; $$);

rollback;
