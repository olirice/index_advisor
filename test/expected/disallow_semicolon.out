begin;
    create extension index_advisor cascade;
NOTICE:  installing required extension "hypopg"
    select index_advisor($$ select 1; $$);
ERROR:  query must not contain a semicolon
CONTEXT:  PL/pgSQL function index_advisor(text) line 15 at RAISE
rollback;
